compute_tools/src/http/openapi_spec.yaml

```diff
@@ -229,6 +268,67 @@ components:
         num_cpus:
           type: integer
 
+    SchemaObjects:
+      type: object
+      description: Schema Databases and Roles
+      required:
+        - roles
+        - databases
+      properties:
+        roles:
+          type: array
+          items:
+            $ref: "#/components/schemas/SchemaRole"
+        databases:
+          type: array
+          items:
+            $ref: "#/components/schemas/SchemaDatabase"
+
+    SchemaDatabase:
+      type: object
+      description: Schema Database
+      required:
+        - name
+        - owner
+      properties:
+        name:
+          type: string
+        owner:
```

Comment:
Let's also specify that we return these attributes
```
    #[serde(skip_deserializing, default)]
    pub restrict_conn: bool,
    #[serde(skip_deserializing, default)]
    pub invalid: bool,
```
with database (I think we should)

Reply:
done